awscrt/io.py

```diff
@@ -39,6 +40,16 @@ def init_logging(log_level, file_name):
     _awscrt.init_logging(log_level, file_name)
 
 
+"""
+Static Defaults (singletons - or as close as possible in Python)
+
+Will be garbage collected when program ends or module is unloaded
+"""
+_static_client_bootstrap = None
+_static_default_host_resolver = None
+_static_event_loop_group = None
```

Comment:
trivial: one of these has the word "default" in it but the others don't. be consistent one way or the other